File: Aula2/Java/Item.java
class Item {
    private String nome;
    private int bonus;
    private String atributo;

    public Item(String nome, int bonus, String atributo) {
        this.nome = nome;

        if (bonus < 0) {
            throw new IllegalArgumentException("O bônus não pode ser negativo.");
        }

        this.bonus = bonus;
        this.atributo = atributo;
    }

    public String getNome() {
        return nome;
    }

    public int getBonus() {
        return bonus;
    }

    public String getAtributo() {
        return atributo;
    }

    public String descricao() {
        return nome + " (+" + bonus + " " + atributo + ")";
    }
    @Override
    public String toString() {
        return descricao();
}
}

File: Aula2/Python/Item.py
class Item:
    def __init__(self, nome, bonus, atributo):
        self.nome = nome

        if bonus < 0:
            raise ValueError("O bônus não pode ser negativo.")

        self.bonus = bonus
        self.atributo = atributo

    def descricao(self):
        return f"{self.nome} (+{self.bonus} {self.atributo})"

    def __str__(self):
        return self.descricao()

    def __repr__(self):
        return self.descricao()